pkgremove

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+branch=$1
+pkgs=$(pkgcheck "${branch}")
+apt-get update
+apt-get install $pkgs
```

Comment:
This should be `remove`, I guess?

Reply:
uh no, it should install, we're not actually removing packages, we're downgrading packages, originally we were running apt full-upgrade here too